<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1.1 Illumina SE | Bioinformatique pour la pharmacie</title>
  <meta name="description" content="1.1 Illumina SE | Bioinformatique pour la pharmacie" />
  <meta name="generator" content="bookdown 0.37 and GitBook 2.6.7" />

  <meta property="og:title" content="1.1 Illumina SE | Bioinformatique pour la pharmacie" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1.1 Illumina SE | Bioinformatique pour la pharmacie" />
  
  
  

<meta name="author" content="Dorine MERLAT - dorine.merlat@et.unistra.fr" />


<meta name="date" content="2024-02-15" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="lectures-ngs.html"/>
<link rel="next" href="pacbio-hifi.html"/>
<script src="assets/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="assets/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="assets/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="assets/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="assets/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="assets/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="assets/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="assets/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="assets/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="assets/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="assets/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="lectures-ngs.html"><a href="lectures-ngs.html"><i class="fa fa-check"></i><b>1</b> Lectures NGS</a>
<ul>
<li class="chapter" data-level="1.1" data-path="illumina-se.html"><a href="illumina-se.html"><i class="fa fa-check"></i><b>1.1</b> Illumina SE</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="illumina-se.html"><a href="illumina-se.html#raw-data-fastq-file"><i class="fa fa-check"></i><b>1.1.1</b> Raw data: FASTQ file</a></li>
<li class="chapter" data-level="1.1.2" data-path="illumina-se.html"><a href="illumina-se.html#contrôle-qualité-avec-fastqe-et-fastqc"><i class="fa fa-check"></i><b>1.1.2</b> Contrôle qualité avec FastQE et FastQC</a></li>
<li class="chapter" data-level="1.1.3" data-path="illumina-se.html"><a href="illumina-se.html#read-procesing-using-trimmomatic"><i class="fa fa-check"></i><b>1.1.3</b> Read procesing using Trimmomatic</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="pacbio-hifi.html"><a href="pacbio-hifi.html"><i class="fa fa-check"></i><b>1.2</b> PacBio Hifi</a></li>
<li class="chapter" data-level="1.3" data-path="nanopore.html"><a href="nanopore.html"><i class="fa fa-check"></i><b>1.3</b> Nanopore</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="exploration-of-sequence-databases.html"><a href="exploration-of-sequence-databases.html"><i class="fa fa-check"></i><b>2</b> Exploration of sequence databases</a>
<ul>
<li class="chapter" data-level="2.1" data-path="database-exploration-using-entrez.html"><a href="database-exploration-using-entrez.html"><i class="fa fa-check"></i><b>2.1</b> Database exploration using Entrez</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="database-exploration-using-entrez.html"><a href="database-exploration-using-entrez.html#search-by-annotation-filtersadvanced"><i class="fa fa-check"></i><b>2.1.1</b> Search by annotation (Filters/Advanced)</a></li>
<li class="chapter" data-level="2.1.2" data-path="database-exploration-using-entrez.html"><a href="database-exploration-using-entrez.html#taxonomy-database"><i class="fa fa-check"></i><b>2.1.2</b> Taxonomy database</a></li>
<li class="chapter" data-level="2.1.3" data-path="database-exploration-using-entrez.html"><a href="database-exploration-using-entrez.html#sequence-retrieval-with-an-access-number"><i class="fa fa-check"></i><b>2.1.3</b> Sequence retrieval with an access number</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="search-for-protein-sequences-in-uniprotkb-www.uniprot.html"><a href="search-for-protein-sequences-in-uniprotkb-www.uniprot.html"><i class="fa fa-check"></i><b>2.2</b> Search for protein sequences in UniProtKB (www.uniprot.org)</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="search-for-protein-sequences-in-uniprotkb-www.uniprot.html"><a href="search-for-protein-sequences-in-uniprotkb-www.uniprot.html#use-of-fields-advanced-search"><i class="fa fa-check"></i><b>2.2.1</b> Use of fields (Advanced Search)</a></li>
<li class="chapter" data-level="2.2.2" data-path="search-for-protein-sequences-in-uniprotkb-www.uniprot.html"><a href="search-for-protein-sequences-in-uniprotkb-www.uniprot.html#example-of-a-swissprot-entry-muc1_human"><i class="fa fa-check"></i><b>2.2.2</b> Example of a SwissProt entry: MUC1_HUMAN</a></li>
<li class="chapter" data-level="2.2.3" data-path="search-for-protein-sequences-in-uniprotkb-www.uniprot.html"><a href="search-for-protein-sequences-in-uniprotkb-www.uniprot.html#function-go-annotations"><i class="fa fa-check"></i><b>2.2.3</b> Function – GO annotations</a></li>
<li class="chapter" data-level="2.2.4" data-path="search-for-protein-sequences-in-uniprotkb-www.uniprot.html"><a href="search-for-protein-sequences-in-uniprotkb-www.uniprot.html#structure"><i class="fa fa-check"></i><b>2.2.4</b> Structure</a></li>
<li class="chapter" data-level="2.2.5" data-path="search-for-protein-sequences-in-uniprotkb-www.uniprot.html"><a href="search-for-protein-sequences-in-uniprotkb-www.uniprot.html#external-links-tab-top-of-the-page"><i class="fa fa-check"></i><b>2.2.5</b> External links tab (top of the page)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="functional-analysis-and-sequence-comparison.html"><a href="functional-analysis-and-sequence-comparison.html"><i class="fa fa-check"></i><b>3</b> Functional analysis and sequence comparison</a></li>
<li class="chapter" data-level="4" data-path="similarity-searches.html"><a href="similarity-searches.html"><i class="fa fa-check"></i><b>4</b> Similarity searches</a></li>
<li class="chapter" data-level="5" data-path="multiple-alignment.html"><a href="multiple-alignment.html"><i class="fa fa-check"></i><b>5</b> Multiple alignment</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Bioinformatique pour la pharmacie</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="illumina-se" class="section level2 hasAnchor" number="1.1">
<h2><span class="header-section-number">1.1</span> Illumina SE<a href="illumina-se.html#illumina-se" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Optionnel: Vous pouvez renommer votre historique (panneau de droite).</p>
<p>Le 1er set de données concerne une étude métagénomique portant sur le microbiome de la cavité orale d’un serpent. Après extraction et fragmentation de l’ADN de l’échantillon, une partie de gène de l’ARNr 16S a été amplifié puis séquencé (Illumina Single End) afin d’identifier les bactéries présentes. Seule une partie des données sera analysée pour des raisons de temps.</p>
<div id="raw-data-fastq-file" class="section level3 hasAnchor" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> Raw data: FASTQ file<a href="illumina-se.html#raw-data-fastq-file" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Importez le fichier FASTQ depuis le serveur zenodo :</li>
</ol>
<p>Copiez l’adresse du lien</p>
<p><a href="https://zenodo.org/record/3977236/files/female_oral2.fastq-4143.gz" class="uri">https://zenodo.org/record/3977236/files/female_oral2.fastq-4143.gz</a></p>
<p>Dans Galaxy, utilisez le bouton <em>Upload Data</em> (en haut à gauche) puis <em>Paste/Fetch dat</em>a, copiez le lien et lancez (<em>start</em>).</p>
<ol start="2" style="list-style-type: decimal">
<li>Regardez le fichier (<em>eye icon</em>).</li>
</ol>
<blockquote>
<p>❓ Quelles sont les caractéristiques du format FASTQ ?</p>
</blockquote>
<blockquote>
<p>❓ Sachant que les scores sont encodés selon le système SANGER/Illumina 1.8+ (<a href="https://support.illumina.com/help/BaseSpace_Sequence_Hub_OLH_009008_2/Source/Informatics/BS/QualityScoreEncoding_swBS.htm" class="uri">https://support.illumina.com/help/BaseSpace_Sequence_Hub_OLH_009008_2/Source/Informatics/BS/QualityScoreEncoding_swBS.htm</a>), quelle est la valeur du score de la plupart des bases au début du 1er read ? Est-ce satisfaisant?</p>
</blockquote>
</div>
<div id="contrôle-qualité-avec-fastqe-et-fastqc" class="section level3 hasAnchor" number="1.1.2">
<h3><span class="header-section-number">1.1.2</span> Contrôle qualité avec FastQE et FastQC<a href="illumina-se.html#contrôle-qualité-avec-fastqe-et-fastqc" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Ouvrir le FASTQ Quality Control toolbox (panneau de gauche).</p>
<div id="fastqe" class="section level4 hasAnchor" number="1.1.2.1">
<h4><span class="header-section-number">1.1.2.1</span> FastQE<a href="illumina-se.html#fastqe" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Explorer la qualité de vos séquences avec FASTQE. “Mean scores” correspond à la moyenne des scores calculées sur toutes les lectures à chaque position. Les scores sont représentés par des emojis (classés de 0 à 41) :</p>
<p>🚫❌👺💔🙅👾👿💀👻🙈🙉🙊🐵😿😾🙀💣🔥😡💩⚠️
😀😅😏😊😙😗😚😃😘😆😄😋😄😝😛😜😉😁😄😎😍</p>
<blockquote>
<p>❓ Quelle est la tendance générale pour ce dataset?</p>
</blockquote>
</div>
<div id="fastqc" class="section level4 hasAnchor" number="1.1.2.2">
<h4><span class="header-section-number">1.1.2.2</span> FastQC<a href="illumina-se.html#fastqc" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Evaluez plus en détail la qualité des lectures avec FastQC (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="uri">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>. Voir FastQC website pour l’aide et des exemples d’analyse.</p>
<blockquote>
<p>❓ <em>Per Base Sequence Quality</em>: Quelle est la longueur des lectures ? Comment évolue la qualité le long des lectures ?</p>
</blockquote>
<blockquote>
<p>❓ <em>Per sequence quality scores</em>: Que vous apprend ce graphe ?</p>
</blockquote>
<blockquote>
<p>❓ <em>Per base sequence content</em>: Dans le cas d’un séquençage de génome (sans amplification), à quoi devrait ressembler ce graphe ? Le résultat obtenu est-il normal dans notre cas ?</p>
</blockquote>
<blockquote>
<p>❓ <em>Adapter content</em>: Quel adaptateur est détecté ?</p>
</blockquote>
</div>
</div>
<div id="read-procesing-using-trimmomatic" class="section level3 hasAnchor" number="1.1.3">
<h3><span class="header-section-number">1.1.3</span> Read procesing using Trimmomatic<a href="illumina-se.html#read-procesing-using-trimmomatic" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Utilisez Trimmomatic (section FASTQ Quality control) pour améliorer la qualité des lectures :</p>
<ul>
<li><p><em>Perform initial IllUMINACLIP step? Yes, select Nextera</em></p></li>
<li><p><em>Sliding window trimming (proposed by default)</em></p></li>
<li><p>Ajouter une nouvelle opération Trimmomatic : <em>Drop reads below a specific length (MINLEN): 20</em></p></li>
</ul>
<blockquote>
<p>❓ Quel pourcentage de lectures a été éliminé (voir fichier log) ?</p>
</blockquote>
<blockquote>
<p>❓ Refaites une analyse FastQC sur les lectures après traitement et comparez les résultats aux données brutes. Le résultat est-il acceptable ?</p>
</blockquote>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="lectures-ngs.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="pacbio-hifi.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="assets/gitbook-2.6.7/js/app.min.js"></script>
<script src="assets/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="assets/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="assets/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="assets/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="assets/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="assets/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="assets/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
},
"toolbar": {
"position": "static"
}
});
});
</script>

</body>

</html>
